src/envs/stick_point.py

```diff
@@ -59,22 +59,19 @@ def __init__(self) -> None:
         # Predicates
         self._Touched = Predicate("Touched", [self._point_type],
                                   self._Touched_holds)
-        self._InContactStickPoint = Predicate(
-            "InContactStickPoint", [self._stick_type, self._point_type],
-            self._InContact_holds)
-        self._InContactRobotPoint = Predicate(
-            "InContactRobotPoint", [self._robot_type, self._point_type],
-            self._InContact_holds)
-        self._InContactRobotStick = Predicate(
-            "InContactRobotStick", [self._robot_type, self._stick_type],
-            self._InContact_holds)
+        self._StickAbovePoint = Predicate("StickAbovePoint",
+                                          [self._stick_type, self._point_type],
+                                          self._Above_holds)
+        self._RobotAbovePoint = Predicate("RobotAbovePoint",
+                                          [self._robot_type, self._point_type],
+                                          self._Above_holds)
         self._Grasped = Predicate("Grasped",
                                   [self._robot_type, self._stick_type],
                                   self._Grasped_holds)
         self._HandEmpty = Predicate("HandEmpty", [self._robot_type],
                                     self._HandEmpty_holds)
-        self._NoPointInContact = Predicate("NoPointInContact", [],
-                                           self._NoPointInContact_holds)
+        self._AboveNoPoint = Predicate("AboveNoPoint", [],
```

Comment:
what is this referring to? the stick or the robot?

Reply:
either one